_includes/includes/page-version.php

```diff
@@ -64,16 +64,17 @@ static function WriteSelector() {
       echo "<div>";
 
       foreach($PageVersion->versions as $version) {
+        $found = true;
         $folder = "{$_SERVER['DOCUMENT_ROOT']}{$PageVersion->basePath}/{$version}/";
         if(!file_exists("{$folder}{$file}.php") && !file_exists("{$folder}/{$file}.md")) {
           // We don't have the same content in that version, so point to home page
           if($version == $PageVersion->currentVersion) {
             $text = "Version $version (home page, current version)";
-            $target = "{$PageVersion->basePath}/current-version/";
+            $target = "{$PageVersion->basePath}/current-version/{$PageVersion->subPath}";
```

Comment:
```suggestion
            $target = "{$PageVersion->basePath}/current-version/";
```

I don't think this change should be here, either :grin: